core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -578,11 +382,12 @@ oppia.factory('explorationRightsService', [
         var that = this;
 
         var explorationModeratorRightsUrl = (
-          '/createhandler/moderatorrights/' + explorationData.explorationId);
+          '/createhandler/moderatorrights/' + 
+            ExplorationDataService.explorationId);
```

Comment:
Please ident it exactly below the above line ?

Reply:
I'm sorry if I misunderstood the request, but I removed the indent below line 385. Or should I increase the indent?

Reply:
Yup, right had to indent line 386 to be exactly under line 385 like 

![indent](https://user-images.githubusercontent.com/7807557/32406806-090ee562-c1a5-11e7-948f-f74ec9015d12.jpeg)